src/main/java/buyer/Buyer.java

```diff
@@ -1,26 +1,29 @@
 package buyer;
 
-import lotto.Lotteries;
-import lotto.Lottery;
+import lotto.Lotto;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
 
 public class Buyer {
-    private final Lotteries lotteries = new Lotteries();
-    private final Budget budget;
+    private final List<Lotto> lotteries = new ArrayList<>();
```

Comment:
Lotteries 클래스를 변경해주셨으니 변수명도 함께 바꿔주는건 어떨까요~?

```suggestion
    private final List<Lotto> lottos = new ArrayList<>();
```